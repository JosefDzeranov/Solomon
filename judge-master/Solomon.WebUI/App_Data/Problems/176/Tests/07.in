91 456
0 68 41 28 23 9 29 74 65 64 43 80 30 39 57 28 26 45 44 60 6 94 53 17 45 28 33 27 91 88 70 53 5 98 12 97 79 59 31 8 56 16 37 38 48 72 9 58 7 87 75 60 45 85 21 67 20 29 83 95 75 7 83 76 94 90 80 31 32 59 48 66 18 41 8 90 61 20 69 72 23 74 25 8 18 15 12 19 82 45 82 54 76 36 32 53 33 61 43 57 92 77 31 47 44 14 46 9 81 31 10 99 60 15 76 5 82 79 62 84 82 65 49 87 77 93 89 82 14 55 62 5 76 16 80 46 48 77 64 9 68 8 38 55 40 83 94 65 59 31 71 53 98 59 75 32 33 44 70 50 24 64 94 31 69 9 28 49 35 35 57 9 20 58 20 79 31 93 52 93 99 29 27 82 83 13 7 21 41 53 17 8 12 19 85 9 13 12 86 35 60 49 34 29 74 67 38 45 44 95 5 40 92 76 50 94 41 17 47 60 7 11 12 82 22 53 96 6 72 16 70 24 11 21 38 54 94 97 12 20 75 61 77 45 53 77 16 36 68 90 68 96 64 30 33 53 41 52 12 26 71 37 72 32 30 80 33 76 42 95 5 43 9 31 85 45 33 67 34 45 42 64 37 82 48 73 59 80 87 84 18 46 90 86 49 43 89 49 19 72 48 40 93 72 89 29 92 37 45 88 25 74 23 66 83 11 99 89 84 57 25 53 88 57 82 58 52 22 81 47 55 84 52 62 96 6 70 65 92 23 81 27 40 99 64 43 28 28 7 43 43 99 64 62 51 95 51 41 89 35 76 74 55 97 37 41 26 61 69 81 40 39 97 66 75 98 17 88 92 43 35 28 19 43 8 56 24 90 15 6 55 34 75 61 47 11 71 42 66 28 22 67 58 26 8 9 7 98 86 52 82 63 48 50 5 52 60 17 94 81 21 36 50 68 31 10 64 40 23 69 32 75 27 69 26 67 69 93 39 98 53 25 8 63 24 91 86 29 61 94 51 58 48 84 11 85 0