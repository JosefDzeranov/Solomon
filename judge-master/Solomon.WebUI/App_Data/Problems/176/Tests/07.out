257,258241758242
